### ISSUE TITLE
Deprecate old module names

### ISSUE THREAD

**Issue by maleadt** [JuliaGPU/CuArrays.jl#158]
BLAS has been renamed to CUBLAS. Could be confusing for users because this now points to LinearAlgebra.BLAS.

**Comment by maleadt** [comment]
I guess not, then. https://github.com/JuliaLang/METADATA.jl/pull/18769

**Comment by MikeInnes** [comment]
Sorry, missed this. If you think we should do this feel free to close the tag pr and we can get it in.

**Comment by andreasnoack** [comment]
You can just remove and reapply the tag after merging this PR. Then the METADATA PR will be updated automatically by Attobot.

**Comment by maleadt** [comment]
> If you think we should do this feel free to close the tag pr and we can get it in.

Not necessarily, but I happened to bump into it with some old code and it seemed deprecation-worthy.
On the other hand, we've never exported the internal module names, and I don't immediately find packages that rely on it. Thoughts?

I'll have a quick try, but don't have much time to spend on it.
